UI/ask.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Affordable and professional web design">
+    <meta name="keywords" content="web design, stack overflow, question and answer">
+    <meta name="author" content="Eugene Owak">
+    <title>Stack Overflow-lite | Welcome</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/style.css" />
+    <script src="main.js"></script>
```

Comment:
Place the scripts at the bottom of the body so that the browser can begin rendering sooner to avoid blank screens

Reply:
Thought the best practice is to put them in the header...